broker-cli/broker-daemon-client/index.js

```diff
@@ -36,22 +47,32 @@ class BrokerDaemonClient {
      * @type {String}
      */
     this.address = rpcAddress || this.config.rpcAddress
+    this.certPath = this.config.rpcCert
+    this.disableSsl = this.config.disableSsl
 
     const [host, port] = this.address.split(':')
 
-    // Set a default port if a TLD is used
+    // Set a default port if the port is not specified
     if (!port) {
       this.address = `${host}:${DEFAULT_RPC_PORT}`
     }
 
     this.proto = loadProto(PROTO_PATH)
 
-    // TODO: Change this to use npm instead of a relative path to the daemon
-    // TODO: we will need to add auth for daemon for a non-local address
-    this.adminService = caller(this.address, this.proto.AdminService)
-    this.orderService = caller(this.address, this.proto.OrderService)
-    this.orderBookService = caller(this.address, this.proto.OrderBookService)
-    this.walletService = caller(this.address, this.proto.WalletService)
+    if (this.disableSsl) {
+      // TODO: Eventually allow broker daemon client to use the cli's logger
+      console.warn('disableSsl is set to true. The CLI will try to connect to the daemon without ssl')
+      this.credentials = grpc.credentials.createInsecure()
+    } else {
+      // Go back to the ./broker-cli/certs directory from the current directory
+      this.cert = readFileSync(path.join(__dirname, PROJECT_ROOT_DIR, this.certPath))
```

Comment:
What will happen if the cert doesn't exist at the path?

Reply:
@martineehrlich readFileSync will error on first use OR if the cert is blank, then `createSsl` will yell at you